lesson07/index.js

```diff
@@ -0,0 +1,45 @@
+const imageArray = [
+    {
+        img: "1.png", 
+        alt:"画像1", 
+        text: "ブックマーク"
+    }, 
+    {
+        img: "2.png", 
+        alt:"画像2", 
+        text: "メッセージ"
+    }
+];
+
+const ul = document.getElementById('js-list');
+const loading = document.getElementById('js-loading');
+const fragment = document.createDocumentFragment();
+
+const createLoading = () => {
+    const loadingImg = document.createElement('img');
+    loadingImg.src = 'loading-circle.gif';
+    loading.appendChild(loadingImg);
+};
+
+
+new Promise((resolve) => {
+    createLoading();
+    setTimeout(() => {
+        resolve(imageArray);
+    }, 3000);
+
+}).then((val) => {
+    for (let i = 0; i < val.length; i++) {
+        const li = document.createElement('li');
+        const img = document.createElement('img');
+    
+        li.textContent = val[i].text;
+        img.src = val[i].img;
+        img.alt = val[i].alt;
+    
+        fragment.appendChild(li).appendChild(img);
+    }
+
+    ul.appendChild(fragment);
+    loading.removeChild(loading.firstChild);
```

Comment:
Here, I think I can cut it out to a function.

Reply:
Thank you for review!
I cut it out to a function.

[7955ce7](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/7/commits/7955ce7326f0fb339faf3880bf4364d0c62c2c30)